app/assets/main/react_components/footprint_form/FootprintForm.jsx

```diff
@@ -1,7 +1,8 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
```

Comment:
Verkar inte användas, eller missuppfattar jag?

```suggestion
import React, { useState } from 'react';
```